nautobot/docs/index.md

```diff
@@ -57,6 +57,10 @@ Nautobot is built on the [Django](https://djangoproject.com/) Python Web framewo
 | Task queuing       | Redis/Celery/django-rq          |
 | Live device access | NAPALM                          |
 
+The low level diagram displays how data travels through Nautobot's application stack.
+
+![Application stack diagram](./media/nautobot_application_stack_low_level.png "Application stack diagram")
```

Comment:
- Suggest labeling **Celery** and **Redis Queue** blocks as *Task Queue* 
- Between **Celery**/**Redis-Queue** blocks and the **Nautobot Task Worker** block, insert a **Redis** block, labeled as *Message Transport*
- Label **Celery Beat** block as *Periodic Task Scheduling*